### PR TITLE
357

### PR DESCRIPTION
**Pull request by dgarus**
Issue [#357](https://github.com/yegor256/jpeek/issues/357)

We cannot remove the 'since' tag because it is mandatory for the current check style policy.

Additionally, I've created the [issue](https://github.com/teamed/qulice/issues/1071) in qulice. 
That is why I didn't remove the JavadocTagsCheck tag from Base.java and DefaultBase.java.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#376](https://github.com/yegor256/jpeek/issues/376) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/f7c98956-ce48-48d9-a9f0-6e692055425c); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by paulodamaso** [comment]
@0crat assign @fanifieiev

**Comment by dgarus** [comment]
@fanifieiev Done.

**Comment by 0crat** [comment]
> @0crat assign @fanifieiev ([here](https://github.com/yegor256/jpeek/issues/376#issuecomment-585874679))

@paulodamaso @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) not enough funds available in the project, can't set budget of job [#376](https://github.com/yegor256/jpeek/issues/376), see [§21](http://www.zerocracy.com/policy.html#21); @fanifieiev[/z](https://www.0crat.com/u/fanifieiev) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

File: src/main/java/org/jpeek/Main.java [hunk 1]
```diff
@@ -39,15 +39,12 @@
  *
  * <p>There is no thread-safety guarantee.
  *
- * @author Yegor Bugayenko (yegor256@gmail.com)
- * @version $Id$
  * @since 0.1
  * @todo #308:30min Due to introduce new version of qulice 0.18.17 there were
```
Comment (fanifieiev) [inline src/main/java/org/jpeek/Main.java]: @dgarus I think you need to remove this todo, before the PR is merged.

Comment (dgarus) [reply src/main/java/org/jpeek/Main.java]: Ok

**Comment by dgarus** [comment]
@fanifieiev Done. 
Thanks!

**Comment by fanifieiev** [comment]
@paulodamaso We are good to merge.

**Comment by 0crat** [comment]
@fanifieiev[/z](https://www.0crat.com/u/fanifieiev) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job [#376](https://github.com/yegor256/jpeek/issues/376) is not in the agenda of @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), can't retrieve data and time of add

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@26cdd6e8d81d, thread: pool-717-thread-1
com.jcabi.github.RtValuePagination$Items[181] java.lang.IllegalStateException: java.io.IOException: failed after 4 attempts
com.jcabi.http.wire.RetryWire[95] java.io.IOException: failed after 4 attempts
```

[0.51.6](https://github.com/zerocracy/farm): Issue: yegor256/jpeek#376, Action: `created`

**Comment by 0crat** [comment]
Job [#376](https://github.com/yegor256/jpeek/issues/376) is not in the agenda of @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), can't inspect

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by sereshqua** [comment]
@fanifieiev please make sure you will find at least 3 issues during next CR, thanks :)

**Comment by sereshqua** [comment]
@fanifieiev ping

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by fanifieiev** [comment]
@sereshqua ok, got it

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**